scripts/reddit.js

```diff
@@ -0,0 +1,78 @@
+// Description:
+//   Grab top non-stickied post of a subreddit.
+//
+// Dependencies:
+//   reddit
+//
+// Configuration:
+//   HUBOT_REDDIT_USERNAME
+//   HUBOT_REDDIT_PASSWORD
+//   HUBOT_REDDIT_APP_ID
+//   HUBOT_REDDIT_APP_SECRET
+//
+// Commands:
+//   hubot reddit [subreddit] - Displays top non-stickied post of [subreddit], defaults to r/popular
+//
+// Notes:
+//   None.
+//
+// Author:
+//   https://github.com/davidscholberg
+
+const Reddit = require('reddit');
+
+function configureReddit() {
+  if (
+    typeof process.env.HUBOT_REDDIT_USERNAME === 'undefined' ||
+    typeof process.env.HUBOT_REDDIT_PASSWORD === 'undefined' ||
+    typeof process.env.HUBOT_REDDIT_APP_ID === 'undefined' ||
+    typeof process.env.HUBOT_REDDIT_APP_SECRET === 'undefined'
+  ) {
+    console.error(`error: missing one or more reddit configuration values`);
+    return null;
+  }
```

Comment:
Seems like it's getting to be time to use something like https://www.npmjs.com/package/dotenv. Not a blocker for this PR though.

Reply:
Totally agree with you here.